### PR TITLE
Replace print statements with logging using mozlog

### LINKED ISSUE #26: Replace print statements with a real logging mechanism
We should not use print statements but make use of a global logger instead. We could expand the Mozmill logger to be testrun wide available.

### PR DESCRIPTION
**Pull request by Ethcelon**
@whimboo 
This is to fix #26

### CODE REVIEW

**Comment by Ethcelon** [comment]
Will this do? @whimboo

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
         'mozfile >= 1.0',
         'mozinstall >= 1.7',
         'mozmill == 2.1-dev',
-        'mozversion >= 0.7'
+        'mozversion >= 0.7',
+        'mozlog >= 2.9'
```
Comment (whimboo) [inline setup.py]: Give that an alphabetical order too and we will be ready for merging your PR!

**Comment by Ethcelon** [comment]
I've changed the alphabetical ordering

**Comment by whimboo** [comment]
This PR has been merged as https://github.com/mozilla/mozmill-automation/commit/1d856c5ada96f3cacf1772aacabeaee5d9714eb7. I did a small change to the logger name, given that it was reporting itself as `root`. Given that mozversion has its own name, we should better also use `mozmill-automation` as the logger name.

Thanks @Ethcelon for working on this issue!

**Pull request closed without merging**